### PR TITLE
Add functional conformance suite page

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
This PR is to add the UK functional conformance suite page.
This will fix https://github.com/wso2/docs-open-banking/issues/686.

### CODE REVIEW

File: en/docs/assets/attachments/endpoint_configuration_for_accounts_API.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            {
+              "method": "GET",
+              "path": "/transactions"
+            }
+          ]
+        }
+      ]
+    }
+  }
+
```
Comment (shadhini) [inline en/docs/assets/attachments/endpoint_configuration_for_accounts_API.json:136]: let's add a new line here, please check other places as well

**Comment by DinithiDiaz** [comment]
Please find a locally built copy of the page for reference:

[functional-conformance-suite.pdf](https://github.com/wso2/docs-ob-toolkit-uk/files/10999324/localhost-8001-conformance-functional-conformance-suite-2023-03-17-13_54_36.pdf)

**Review by shadhini — CHANGES_REQUESTED** [review]

File: en/docs/conformance/functional-conformance-suite.md [hunk 2]
```diff
@@ -0,0 +1,625 @@
+!!!info
+    WSO2 Open Banking 3.0.0 UK Toolkit 1.0.0 supports Functional Conformance Suite  v1.7.1.
```
[the remark this answers is not in the document]
Comment (Ashi1993) [reply en/docs/conformance/functional-conformance-suite.md:2]: Lets comment this for now till we finalize the failing test cases

File: en/docs/assets/attachments/endpoint_configuration_for_accounts_API.json (hunk 1 above)
Comment (DinithiDiaz) [reply en/docs/assets/attachments/endpoint_configuration_for_accounts_API.json]: Fixed via 496494d6e96b06aecae1ee51f1cda3fc6d53253c.

File: en/docs/conformance/functional-conformance-suite.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (DinithiDiaz) [reply en/docs/conformance/functional-conformance-suite.md]: Fixed via 186da86db64dd6e6afa2793c507ead9f036f9e27.

File: en/docs/assets/attachments/endpoint_configuration_for_cof_API.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      "discoveryItems": [
+        {
+          "apiSpecification": {
+            "name": "Confirmation of Funds API Specification",
+            "url": "https://openbankinguk.github.io/read-write-api-site3/v3.1.6/profiles/confirmation-of-funds-api-profile.html",
+            "version": "v3.1.6",
+            "schemaVersion": "https://raw.githubusercontent.com/OpenBankingUK/read-write-api-specs/v3.1.6/dist/swagger/confirmation-funds-swagger.json",
+            "manifest": "file://manifests/ob_3.1_cbpii_fca.json"
+          },
+          "openidConfigurationUri": "https://localhost:9446/oauth2/token/.well-known/openid-configuration",
```
[the remark this answers is not in the document]
Comment (DinithiDiaz) [reply en/docs/assets/attachments/endpoint_configuration_for_cof_API.json]: Fixed via d8f10ecd48983a4e70c6a3cca5018116a7f2e004.

**Review by Ashi1993 — APPROVED** [review]

**Review by shadhini — APPROVED** [review]

**Pull request merged**